src/compute-client/src/controller.rs

```diff
@@ -365,7 +366,7 @@ impl<T: ComputeControllerTimestamp> ComputeController<T> {
         result
     }
 
-    /// Returns `true` iff all (non-transient) collections have their write
+    /// Returns `true` if all non-transient, non-excluded collections have their write
```

Comment:
I removed all the "iff"s because they were confusing in connection with the zero-replicas exception. "Returns `true` if, _and only if_, all collections are caught up... Btw, it also returns true when there are no replicas!"